src/OpenLoco/src/Map/Tile.cpp

```diff
@@ -75,9 +75,7 @@ size_t Tile::size()
 
 TileElement* Tile::operator[](size_t i)
 {
-#if DEBUG
     assert(i < size());
```

Comment:
Looks like this could affect performance by quite a bit

Reply:
Not saying it should be removed at this stage, it might still come in quite handy, just mentioning it for a future revision

Reply:
If it did it hasn't been an issue to anyone so far since DEBUG has always been defined... I didn't mean to keep this deletion it was just from playing about fixing warnings, I will probably revert it.